spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestViews.java

```diff
@@ -1149,6 +1149,159 @@ public void createViewWithSubqueryExpressionUsingGlobalTempView() {
                 "because it references to the temporary object global_temp.%s", globalTempView));
   }
 
+  @Test
+  public void describeView() {
+    String viewName = "describeView";
+
+    sql("CREATE VIEW %s AS SELECT id, data FROM %s WHERE id <= 3", viewName, tableName);
+    assertThat(sql("DESCRIBE %s", viewName))
+        .containsExactlyInAnyOrder(row("id", "int", ""), row("data", "string", ""));
```

Comment:
I think `InAnyOrder` is incorrect. Don't we want these to be in the schema order?

Reply:
I agree, this should be `containsExactly()`